src/infra/runtime/runtimeCmdRepo.go

```diff
@@ -21,7 +21,7 @@ func NewRuntimeCmdRepo() *RuntimeCmdRepo {
 	return &RuntimeCmdRepo{runtimeQueryRepo: RuntimeQueryRepo{}}
 }
 
-func (repo *RuntimeCmdRepo) restartPhp() error {
+func (repo *RuntimeCmdRepo) RestartPhp() error {
```

Comment:
Look at the code man, it's using services cmd. If you really needed to restart the php-ws, you could have used servicesCmdRepo, not made an auxiliary method a public one and exported it to the interface. That's wild.